setup.cfg

```diff
@@ -50,14 +50,16 @@ docs =
   towncrier
 
 [options.package_data]
-sunpy.data = sunpyrc, test/**
+sunpy.data = sunpyrc, test/*, test/*/*
 sunpy.database = tests/test_table.txt
 sunpy.io = special/asdf/schemas/sunpy.org/sunpy/*/*.yaml, special/asdf/schemas/sunpy.org/sunpy/*/*/*.yaml
 sunpy.tests = figure_hashes_py36.json
 
 [options.entry_points]
 asdf_extensions =
     sunpy = sunpy.io.special.asdf.extension:SunpyExtension
+pytest11 =
+    asdf = asdf.tests.schema_tester
```

Comment:
has this made it so you have to have asdf installed to run our tests?

Reply:
Yes

Reply:
Not the end of the world I suppose.